controls/cis_rhel8.yml

```diff
@@ -1320,270 +1320,6 @@ controls:
     related_rules:
       - nftables_ensure_default_deny_policy
 
-  - id: 3.4.1.5
-    title: Ensure firewalld default zone is set (Automated)
-    levels:
-      - l1_server
-      - l1_workstation
-    status: automated
-    rules:
-      - set_firewalld_default_zone
```

Comment:
We should remove the `cis@rhel8` references on these rules no longer mentioned in the control file.

Reply:
Already removed

Reply:
Unfortunately I can still see `cis@rhel8:` references in rule no longer mentioned in control file.
Basically are rules removed by this commit https://github.com/ComplianceAsCode/content/pull/11469/commits/8f140f6b6009ff91f6b177df7b76173484c5b88e

We can try to clean-up this after, but doing it now would be good.